### PR TITLE
Fix/tilt terra readiness

### PR DESCRIPTION
**Pull request by evan-gray**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/849)

### CODE REVIEW

**Review by SEJeff — DISMISSED** [review]
Awesome stuff!

My only suggestion is could you pretty please fix your editor to ensure files have trailing `\n`?

File: Dockerfile.client [hunk 1]
```diff
@@ -14,27 +14,26 @@ RUN --mount=type=cache,target=/root/.cache \
...
 ENV PATH="$SOLANA_BIN_PATH:$PATH"
 
-ADD ethereum /usr/src/ethereum
 WORKDIR /usr/src/ethereum
-RUN --mount=type=cache,target=/root/.cache \
-	--mount=type=cache,target=/root/.npm \
-    npm ci
+COPY ethereum/package.json ethereum/package-lock.json ./
+RUN npm ci
+COPY ethereum ./
```
Comment (leoluk) [inline Dockerfile.client:20]: We might want to keep the cache `--mount` to speed up `npm ci` runs whenever package-lock.json changes?

File: Dockerfile.client [hunk 2]
```diff
@@ -14,27 +14,26 @@ RUN --mount=type=cache,target=/root/.cache \
...
 
-ADD clients/token_bridge /usr/src/clients/token_bridge
 WORKDIR /usr/src/clients/token_bridge
-RUN --mount=type=cache,target=/root/.cache \
-	--mount=type=cache,target=/root/.npm \
-	set -xe && \
-	npm ci && \
+COPY clients/token_bridge/package.json clients/token_bridge/package-lock.json ./
+RUN set -xe && \
```
Comment (leoluk) [inline Dockerfile.client:24]: Does `set -xe` do anything for a single statement?

**Review by leoluk — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]

**Review by leoluk — APPROVED** [review]
> Reviewed 27 of 27 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/849)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @evan-gray)

**Pull request merged**